docs/management_api.yml

```diff
@@ -1197,9 +1197,36 @@ paths:
         200:
           description: OK
           schema:
-            type: object
-            properties:
-              $ref: "#/definitions/ArtifactUploadLink"
+            $ref: "#/definitions/ArtifactUploadLink"
+        400:
```

Comment:
how can I get 400 when using this endpoint? isn't it the POST without body?
and, btw, why it's POST and not GET?

Reply:
I updated the swagger documentation reflecting the actual responses from the API.
It's `POST` and not `GET` because you create a "Link" object.